### PR TITLE
chore(integrations): Update docs for LangChain integration

### PR DESCRIPTION
**Pull request by tssweeney**
Docs!

### CODE REVIEW

File: wandb/integration/langchain/wandb_tracer.py [hunk 1]
```diff
@@ -117,15 +117,24 @@ def init(
     ) -> None:
         """Sets up a WandbTracer and makes it the default handler.
 
+        Parameters:
+            run_args: (dict, optional) Arguments to pass to `wandb.init()`. If not provided, `wandb.init()` will be
+                called with no arguments. Please refer to the `wandb.init` for more details.
+                https://github.com/wandb/wandb/blob/9967dde4313aa8faeac4ca50891dde6db7be6342/wandb/sdk/wandb_init.py#L913
```
Comment (kptkin) [inline wandb/integration/langchain/wandb_tracer.py:123]: minor request to maybe use the `wandb.me/...` redirects and/or link to a relevant part of our docs (if available)?

File: wandb/integration/langchain/wandb_tracer.py [hunk 2]
```diff
@@ -152,9 +165,16 @@ def finish() -> None:
     def init_run(self, run_args: Optional[WandbRunArgs] = None) -> None:
         """Initialize wandb if it has not been initialized.
 
+        Parameters:
+            run_args: (dict, optional) Arguments to pass to `wandb.init()`. If not provided, `wandb.init()` will be
+                called with no arguments. Please refer to the `wandb.init` for more details.
+                https://github.com/wandb/wandb/blob/9967dde4313aa8faeac4ca50891dde6db7be6342/wandb/sdk/wandb_init.py#L913
```
Comment (kptkin) [inline wandb/integration/langchain/wandb_tracer.py:171]: same as above

**Review by kptkin — APPROVED** [review]

File: wandb/integration/langchain/wandb_tracer.py (hunk 1 above)
Comment (ngrayluna) [inline wandb/integration/langchain/wandb_tracer.py:123]: This URL might get stale in the future, maybe we can remove it?

File: wandb/integration/langchain/wandb_tracer.py (hunk 2 above)
Comment (ngrayluna) [inline wandb/integration/langchain/wandb_tracer.py:171]: Sam here.

File: wandb/integration/langchain/wandb_tracer.py (hunk 1 above)
Comment (kptkin) [reply wandb/integration/langchain/wandb_tracer.py:123]: great minds lol

**Review by ngrayluna — APPROVED** [review]

**Pull request merged**